src/main/java/com/ays/assignment/model/entity/AssignmentEntity.java

```diff
@@ -100,6 +101,15 @@ public void reserve(String userId, String institutionId) {
         this.institutionId = institutionId;
     }
 
+    public void updateAssignment(AssignmentUpdateRequest updateRequest) {
+        this.description = updateRequest.getDescription();
+        this.firstName = updateRequest.getFirstName();
+        this.lastName = updateRequest.getLastName();
+        this.countryCode = updateRequest.getPhoneNumber().getCountryCode();
+        this.lineNumber = updateRequest.getPhoneNumber().getLineNumber();
+        this.setPoint(updateRequest.getLongitude(), updateRequest.getLatitude());
+    }
```

Comment:
- Metot ismini sadece `update()` yapmamız yeterli olur çünkü bu metot kullanılırken `assignmentEntity.update()` gibi kullanılacak
- Bu metoda `AssignmentUpdateRequest` göndermek ve objeleri bir birine bağlamak yerine, servis katmanında önce `AssignmentUpdateRequest` objesini `AssignmentEntity` objesine maplersek ve daha sonra bu metoda `AssignmentEntity` göndererek update dersek daha okunaklı ve objeleri birbirine bağlamayan bir yapı elde etmiş oluruz.

Reply:
Düzelttim. 
Aynı yapı UserEntity'de bulunduğu için ben de öyle yapmıştım.
![Screenshot from 2023-09-12 16-08-23](https://github.com/afet-yonetim-sistemi/ays-be/assets/116587797/5a1b1abd-d0c0-4ec0-b538-d0dbf2e75ec8)

Reply:
Hallettim.